charts/gatekeeper/templates/gatekeeper-controller-manager-poddisruptionbudget.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-{{- $v1 := .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" -}}
-{{- $v1beta1 := .Capabilities.APIVersions.Has "policy/v1beta1/PodDisruptionBudget" -}}
+{{ $v1 := .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" -}}
+{{ $v1beta1 := .Capabilities.APIVersions.Has "policy/v1beta1/PodDisruptionBudget" -}}
```

Comment:
Please refer to [Contributing to Helm Chart](https://open-policy-agent.github.io/gatekeeper/website/docs/help#contributing-to-helm-chart) for modifying the Helm chart.

Reply:
I've updated the corresponding files. Please have a look.

Reply:
@tberreis you need to remove the changes from `charts`. The changes will be promoted to the root charts directory the next time a Gatekeeper release is cut. You need to limit the changes to `cmd/build/helmify` and `manifest_staging`.

Reply:
Thx @JaydipGabani. Done.